group-generators/helpers/data-providers/atst/interface-schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "AttestationStation",
+  "iconUrl": "",
+  "providerClassName": "AttestationStationProvider",
+  "functions": [
+    {
+      "name": "Get attestation receivers",
+      "functionName": "getAttestations",
+      "countFunctionName": "getAttestationsCount",
+      "description": "Returns all the receivers of a specific attestation on AttestationStation",
+      "args": [
```

Comment:
Also for now the factory support only 1 argument (but there will be very soon the possibility to put 2 args)
So maybe we should wait we implement the mutltiple arguments function before merging your PR

Reply:
Thanks for review! I resolved the other comments.

I'm a bit unsure about this one.
You mentioned Talentlayer as an example on the previous comment, which seems to use multiple args as well.
https://github.com/sismo-core/sismo-hub/blob/main/group-generators/helpers/data-providers/talentlayer/interface-schema.json

Maybe I'm missing something?

Reply:
Yes they have implemented multiple arguments but only the first one is taking into account by the factory.
That is why the type has optional fields, because it used only the first one and all the other arguments are hardcoded for now :)

Reply:
Got it. Thanks!